regions/shapes/circle.py

```diff
@@ -113,21 +113,8 @@ def to_pixel(self, mywcs, mode='local', tolerance=None):
         if tolerance is not None:
             raise NotImplementedError()
 
-        wcsframe = wcs.utils._wcs_to_celestial_frame_builtin(mywcs)
-        center_in_wcsframe = self.center.transform_to(wcsframe)
-
-        xpix, ypix = mywcs.wcs_world2pix(center_in_wcsframe.spherical.lon,
-                                         center_in_wcsframe.spherical.lat, 0)
-
-        if self.radius.unit.physical_type == 'angle':
-            central_pos = coordinates.SkyCoord([mywcs.celestialwcs.crval],
```

Comment:
There is clearly a typo here, but I can't figure out what the author wanted to do with this line